README.md

```diff
@@ -204,15 +204,22 @@ Calculate phonons with a 2x2x2 supercell, after geometry optimization (using the
 janus phonons --struct tests/data/NaCl.cif --supercell 2x2x2 --minimize --arch mace_mp --calc-kwargs "{'model' : 'small'}"
 ```
 
-This will save the Phonopy parameters, including displacements and force constants, to `NaCl-params.yml`, and the calculated band structure to `NaCl-auto-band.yml`, in addition to generating a log file, `phonons.log`, and summary of inputs, `phonons_summary.yml`.
+This will save the Phonopy parameters, including displacements and force constants, to `NaCl-phonopy.yml` and `NaCl-force\_constants.hdf5`, in addition to generating a log file, `phonons.log`, and summary of inputs, `phonons_summary.yml`.
+Additionaly, to calculate band structure and generate `NaCl-auto-bands.yml` use:
+
+```shell
+janus phonons --struct tests/data/NaCl.cif --supercell 2x2x2 --minimize --arch mace_mp --calc-kwargs "{'model' : 'small'}"
+```
+If you need eigenvectors and group velocities written add `--write-full` option. Will generate a much bigger bands file but can be
+used to visualise phonon modes.
```

Comment:
```suggestion

If you need eigenvectors and group velocities written, add the `--write-full` option. This will generate a much larger file, but can be used to visualise phonon modes.
```